help_to_heat/frontdoor/views.py

```diff
@@ -797,6 +799,11 @@ def get_context(self, session_id, *args, **kwargs):
         return {"supplier": supplier}
 
 
+@register_page("shell-warning-page")
+class ShellWarningPageView(PageView):
+    pass
```

Comment:
Why's this just `pass`, rather than getting the supplier like the others?

Reply:
its weird - I can't find any particular reason why the above classes put the supplier into context (since the pages they render are static anyway), so imo we should consolidate one way or the other